windows/generate_cmap4_from_12.py

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env python2
```

Comment:
I used this [sample script](https://github.com/behdad/fonttools/blob/master/Snippets/cmap-format.py) in font tools as inspiration. It's under an MIT style license as well, however I'm not sure what the guideline is for attribution here.